### ISSUE TITLE
Does it make sense to run init_test?

### ISSUE THREAD

**Issue by vitcpp** [postgrespro/pgsphere#54]
Dear All,

What is the purpose of init_test? Do we really need it? This test introduces some difficulties when working with PRs. It is executed when executing gmake test, but gmake installcheck doesn't execute this test.

I propose to remove it.

**Comment by esabol** [comment]
I'm not sure what the intention was.

If you do remove it, would `gmake test` just be an alias for `gmake installcheck`?

**Comment by esabol** [comment]
> There is the difference between test and installcheck rules. gmake installcheck runs tests on the existing installation. gmake test creates a local installation, runs postgresql instance and executes tests on it. These rules play different roles.

Yes, I knew that, or I thought so anyway. What I meant was the intention of `init_test.sql` specifically.

**Comment by vitcpp** [comment]
The init_test goal is to find problems in pgs_*.sql files that are used to initialize the extension. This test creates a single file pg_sphere.test.sql from the contents of pgs_*.sql files with some changes (MODULE_PATH is replaced with the absolute path to local pg_sphere.so) and then runs the file as a regular test. If a problem exists, it can be detected by the test output difference after test execution. I think this test is reasonable because it may help developers detect some problems with newly created functionality as well as when migrating to newer versions of Postgresql. Anyway, it should be improved to avoid expected/init_test.out conflicts when touching independent pgs_*.sql files.

**Comment by esabol** [comment]
Close this issue, @vitcpp ?

**Comment by vitcpp** [comment]
@esabol Yes, thank you. I will close the issue.

**Comment by vitcpp** [comment]
Closed as completed in #79.
